src/css/components/activity_log.scss

```diff
@@ -2,82 +2,91 @@
 @import '../override_vars.scss';
 @import '../vars.scss';
 
-.activity_log-container h1 {
-  font-size: 1.3rem;
-  font-weight: 800;
-  text-transform: uppercase;
-}
-
 .activity_log {
   list-style-type: none;
-  margin-top: 1rem;
   margin-left: 0;
+  margin-top: 1rem;
   padding-left: 0;
   width: 100%;
-
-  li::before {
-    display: none;
-  }
 }
 
 .activity_log-item {
   align-items: center;
-  border-bottom: 1px solid #ccc;
+  border-bottom: 1px solid $color-lightgray;
   cursor: pointer;
   display: flex;
   flex-wrap: wrap;
-  margin: 0;
-  padding-left: 1rem;
-  padding-top: .5rem;
+  margin-bottom: .5rem;
+  margin-top: .5rem;
   padding-bottom: .5rem;
+  padding-left: 1rem;
+  position: relative;
   width: 100%;
-}
 
-.activity_log-item-console p {
-  font-family: monospace;
-  word-wrap: break-word;
-}
+  .activity_log-item_text {
+    position: relative;
+  }
 
-.activity_log-item-error,
-.activity_log-item-warning,
-.activity_log-item-success {
-  border-left-width: 5px;
-  border-left-style: solid;
-}
+  .activity_log-item_text p,
+  .activity_log-item_timestamp {
+    font-family: $font-mono;
+    font-size: $sans-s3;
+    max-width: initial;
+  }
 
-.activity_log-item-error {
-  border-left-color: $color-secondary; // #e31c3d;
+  &:first-child {
+    border-top: 1px solid $color-lightgray;
+    padding-top: .5rem;
+  }
+
+  &.activity_log-item-error,
+  &.activity_log-item-success,
+  &.activity_log-item-warning {
+    .activity_log-item_text::before {
+      border-radius: 3px;
+      border-style: solid;
+      border-width: 2px;
+      content: '';
+      height: 100%;
+      left: -1rem;
+      position: absolute;
+      top: 0;
+    }
+
+    .activity_log-item_text p {
+      font-family: $font-sans;
+      font-weight: 600;
+    }
+  }
 }
 
-.activity_log-item-warning {
-  border-left-color: $color-gold;
+.activity_log-container p {
+  word-wrap: break-word;
 }
 
-.activity_log-item-success {
-  border-left-color: $color-green; //#2e8540;
+.activity_log-item-error .activity_log-item_text::before {
+  border-color: $color-secondary; // #e31c3d;
 }
 
-.activity_log-item:before {
-  content: '';
-  padding: 0;
+.activity_log-item-error .activity_log-item_timestamp {
+  color: $color-secondary;
 }
 
-.activity_log-item {
-  border-bottom: 2px solid #ccc;
+.activity_log-item-warning .activity_log-item_text::before {
+  border-color: $color-gold;
 }
 
-.activity_log-item:first-child {
-  border-top: 2px solid #ccc;
+.activity_log-item-success .activity_log-item_text::before {
+  border-color: $color-green; //#2e8540;
 }
 
-.activity_log-item_text,
+.activity_log-item_text p,
 .activity_log-item_timestamp {
```

Comment:
This ruleset with:

```
activity_log-item_text p,
.activity_log-item_timestamp {
```

is already on line 30. Can the `flex-grow: 1` be moved there?